<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gpi_MainWindow</class>
 <widget class="QMainWindow" name="gpi_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gpi_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <spacer name="gpi_horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>407</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="gpi_label_author">
      <property name="text">
       <string>=Галанин П. И.=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="gpi_label_title">
      <property name="text">
       <string>=gpi_ootpisp5_lab3_option5_task6=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="gpi_label_file2">
        <property name="text">
         <string>gpi_g.txt Файл с чётными числами</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="gpi_label_file1">
        <property name="text">
         <string>gpi_f.txt Файл с целыми числами</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="gpi_plainTextEdit_file1"/>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="gpi_plainTextEdit_file2"/>
      </item>
      <item row="1" column="2">
       <widget class="QPlainTextEdit" name="gpi_plainTextEdit_file3"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="gpi_label_file3">
        <property name="text">
         <string>gpi_h.txt Файл с нечётными числами</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLineEdit" name="gpi_lineEdit_path">
      <property name="text">
       <string>:/@/gpi_f.txt</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="gpi_label_path">
      <property name="text">
       <string>gpi_ Путь до файла</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
